### ISSUE TITLE
Doesn't catch emojis from Web Component

### ISSUE THREAD

**Issue by terinjokes** [paulcbetts/github-emoji-switcher#2]
It looks like GitHub has started to use an emoji Web Component in some situations, such as Issues and Pull Requests. 😢 

![](https://share.terinstock.com/2016-04-13_17-46-02.png)

**Comment by anaisbetts** [comment]
Yeah, this plugin needs to be updated for sure - wanna hack on it?

**Comment by terinjokes** [comment]
Sure, I can give it a shot.
